doc/changelog.md

```diff
@@ -115,6 +115,21 @@
       m = measure(0, postselect=1)
       return qml.expval(qml.PauliZ(0))
   ```
+* Add support for reset option in mid-circuit measurements.
```

Comment:
````suggestion
  ```

* Add support for reset option in mid-circuit measurements.
````

Reply:
Sorry for that. Done.